test/lib/cql_test_env.cc

```diff
@@ -654,6 +656,11 @@ class single_node_cql_env : public cql_test_env {
 
             db.invoke_on_all(&replica::database::start).get();
 
+            task_manager.start().get();
+            auto stop_task_manager = defer([&task_manager] {
+                task_manager.stop().get();
+            });
+
```

Comment:
Is it used in this series?
For testing only the api at this point, I'm not sure if it's needed.